.ddev/config.toml

```diff
@@ -97,6 +97,8 @@ pymongo = ['Apache-2.0']
 requests-kerberos = ['ISC']
 # https://github.com/requests/requests-ntlm/blob/master/LICENSE
 requests-ntlm = ['ISC']
+# https://gitlab.com/thelabnyc/requests-unixsocket2/-/blob/master/LICENSE
```

Comment:
Would you mind adding a comment that explains that this is temporary and that we should remove it once the relevant issue or PR in the requests-unixsocket2 repo is complete?